### PR TITLE
FIx: Fix orcid

### PR DESCRIPTION
**Pull request by kmaid**

### CODE REVIEW

File: packages/model/src/user.ts [hunk 1]
```diff
@@ -142,7 +142,7 @@ export interface UserPatchRequest {
   skillsDescription?: string;
   questions?: string[];
   teams?: Pick<UserTeam, 'id' | 'approach' | 'responsibilities'>[];
-  social?: UserSocialLinks;
+  social?: Omit<UserSocialLinks, 'orcid'>;
```
Comment (peterstarling) [inline packages/model/src/user.ts:145]: why dont you remove orcid from the `UserSocialLinks` instead? I don't think it's actually being used anywhere

**Review by peterstarling — CHANGES_REQUESTED** [review]

File: packages/model/src/user.ts (hunk 1 above)
Comment (kmaid) [reply packages/model/src/user.ts:145]: Because it is being used

Comment (peterstarling) [reply packages/model/src/user.ts:145]: where?

Comment (kmaid) [reply packages/model/src/user.ts:145]: Social Icons & Social modal & fixtures tests etc

Comment (peterstarling) [reply packages/model/src/user.ts:145]: yeah i see but how does that make any sense - logically - if orcid isn't actually available there? like.. unless this is meant to be a hotfix i would go through those fixtures and remove all the references

Comment (kmaid) [reply packages/model/src/user.ts:145]: It was supposed to move into socialLinks. It is available there to read but not to write it would seem

**Review by peterstarling — APPROVED** [review]

**Pull request merged**